addon/components/data-col.js

```diff
@@ -63,20 +62,25 @@ export default Component.extend({
       if (value && stylingProperty) {
         let key    = stylingProperty.key || dasherize(property);
         let suffix = stylingProperty.suffix || '';
-
+        if (typeof (value) === 'string') {
+          suffix = '';
+        }
         styles.push(`${key}: ${value}${suffix};`);
       }
     }
 
     return htmlSafe(styles.join(' '));
   }),
-
   value: computed('_values', '_property.{format}', function() {
     let values = get(this, '_values');
     let format = get(this, '_property.format');
 
     return format.apply(null, values);
   }),
+  title: computed('_values', function() {
+    let title = get(this, '_values');
```

Comment:
I think the title must be the same as the value? Then use `value` instead of `_values`